Source/WebCore/dom/PointerEvent.h

```diff
@@ -56,6 +56,8 @@ class PointerEvent final : public MouseEvent {
         long tiltX { 0 };
         long tiltY { 0 };
         long twist { 0 };
+        double altitudeAngle { piOverTwoDouble / 2 };
```

Comment:
`piOverFourDouble`?

Reply:
We just want `piOverTwoDouble` here, actually.